core/templates/dev/head/components/forms/schema_editors/SchemaBasedUnicodeEditorDirective.js

```diff
@@ -115,6 +116,12 @@ oppia.directive('schemaBasedUnicodeEditor', [
             return $sce.trustAsHtml(
               $filter('convertUnicodeWithParamsToHtml')($scope.localValue));
           };
+
+          $scope.displayPopUpMessage = function() {
```

Comment:
This function should be called when you want to start the countdown timer. If you want that timer to start when the directive loads, then maybe you don't even need the outer function -- you can just do $timeout(...) directly in the directive's controller.